### PR TITLE
Create BMI Calculator

### PR DESCRIPTION
**Pull request by muradha**

### CODE REVIEW

File: Projects/BMI Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Projects
```
Comment (sree-hari-s) [inline Projects/BMI Calculator/README.md:1]: Project Name and a brief description how you did it? 
```suggestion
# Projects
```

File: Projects/BMI Calculator/BMICalculator [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+def main():
+  """Prompts the user to enter their height and weight, calculates their BMI, and prints the result."""
+
+  height = float(input("Enter your height in meters: "))
+  weight = float(input("Enter your weight in kilograms: "))
+
+  bmi = calculate_bmi(height, weight)
+
+  print(f"Your BMI is {bmi}.")
```
Comment (sree-hari-s) [inline Projects/BMI Calculator/BMICalculator:24]: rounding the bmi to 2 decimal would be nice.

**Comment by sree-hari-s** [comment]
Just a doubt why there is no extension `.py` for the file you created BMICalculator?

**Comment by muradha** [comment]
# Projects
The Python BMI Calculator project is a simple yet essential tool designed to calculate Body Mass Index (BMI) for users. BMI is a widely used indicator of a person's overall health based on their height and weight. This project allows users to input their height and weight, and it computes their BMI, providing valuable information about their body composition.

**Comment by sree-hari-s** [comment]
> # Projects
> The Python BMI Calculator project is a simple yet essential tool designed to calculate Body Mass Index (BMI) for users. BMI is a widely used indicator of a person's overall health based on their height and weight. This project allows users to input their height and weight, and it computes their BMI, providing valuable information about their body composition.

update this in a readme.md file and you can raise a PR again.

**Pull request closed without merging**